### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by jenssegers**
Added some content and examples.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Contributing to the API
 
-The `master` branch is the published branch on Apiary. `v1` serves as an api-blueprint formatted copy of the legacy documentation. 
+The `master` branch is automatically published on Apiary if there are not syntax errors. The `v1` branch serves as an api-blueprint formatted copy of the legacy documentation.
```
Comment (Brammm) [inline CONTRIBUTING.md]: B: `no syntax errors`, typo.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,62 +16,59 @@ The `master` branch is the published branch on Apiary. `v1` serves as an api-blu
...
 ### RPC
 
-We are creating an *RPC*-style API.  
-We are often looking at how [Slack](https://api.slack.com/methods) approaches this.
+We are creating an *RPC*-style API, similar to the [Slack API](https://api.slack.com/methods).
 
-```
-/object.someAction
-```
+These are common actions we use. *Note*: We choo
```
Comment (Brammm) [inline CONTRIBUTING.md]: B: sentence cut off?

Comment (jenssegers) [inline CONTRIBUTING.md]: Choo Choo

File: CONTRIBUTING.md (hunk 1 above)
Comment (jenssegers) [inline CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,62 +16,59 @@ The `master` branch is the published branch on Apiary. `v1` serves as an api-blu
...
+We are creating an *RPC*-style API, similar to the [Slack API](https://api.slack.com/methods).
 
-```
-/object.someAction
-```
+These are common actions we use. *Note*:
+
+ - **resource.list** - Get a list objects
+ - **resource.info** - Information about a single object
+ - **resource.create** - Create a new object
```
Comment (stivni) [inline CONTRIBUTING.md]: PP : I would **not** force our developers into using CRUD naming 😛

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,62 +16,59 @@ The `master` branch is the published branch on Apiary. `v1` serves as an api-blu
...
-### GET vs POST
+### GET vs. POST
 
 For actions, we require people to `POST` with a `json` body.
 For retrieving data, we suggest to `GET` with query parameters.
 For sake of simplicity, we also allow you to `POST` with your query as a json body.
 
-_example needed_
+```json
+POST deals.list
```
Comment (stivni) [inline CONTRIBUTING.md]: NTH : add the same example in GET format?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -16,62 +16,59 @@ The `master` branch is the published branch on Apiary. `v1` serves as an api-blu
...
+- The objects are camelcased
+- The actions are camelcased
+- The parameters or object properties are snakecased
 
 ```
 /someObject.someAction?some_parameter=value
 ```
 
-### Response structue
+### Format specifications
```
Comment (stivni) [inline CONTRIBUTING.md]: B : This will conflict with my changes...

File: CONTRIBUTING.md (hunk 3 above)
Comment (jenssegers) [inline CONTRIBUTING.md]: No but if we need them, we need somewhat of a convention as well.

**Pull request merged**